server/game/gamesteps/taxationphase.js

```diff
@@ -44,6 +44,8 @@ class TaxationPhase extends Phase {
         this.game.addAlert('endofround', 'End of round {0}', this.game.round);
         this.game.addMessage(playerPower);
         this.game.addAlert('startofround', 'Round {0}', this.game.round + 1);
+
+        this.game.checkForTimeExpired();
```

Comment:
Per tournament rules, should the game end at the beginning of the next round or at the end of the current round? If the latter, this check should come before the `startofround` alert.